docs/book/src/plugins/available/grafana-v1-alpha.md

```diff
@@ -1,17 +1,23 @@
 # Grafana Plugin (`grafana/v1-alpha`)
 
-The Grafana plugin is an optional plugin that can be used to scaffold Grafana Dashboards to allow you to check out the default metrics which are exported by projects using [controller-runtime][controller-runtime].
+The Grafana plugin is an optional plugin that can be used to
+scaffold Grafana Dashboards to allow you to check out the
+default metrics which are exported by projects
+using [controller-runtime][controller-runtime].
 
 <aside class="note">
 <h1>Examples</h1>
 
-You can check its default scaffold by looking at the `project-v3-with-metrics` projects under the [testdata][testdata] directory on the root directory of the Kubebuilder project.
+You can check its default scaffold by looking at the `project-v4-with-plugins` projects
+under the [testdata][testdata] directory on the root directory of the Kubebuilder project.
```

Comment:
I would change the wording here. The way is written, it seems you will click on the link and then need to click on `project-v4-with-plugins`. But the link already takes you to the right folder

Reply:
hmm clicking the link does not take me github https://deploy-preview-4185--kubebuilder.netlify.app/testdata/project-v4-with-plugins

Is this a problem with preview page or the link itself?

Reply:
We will not able to use relative paths within. 
Good catcher 🥇 
Fixed